src/acb_theta.h

```diff
@@ -0,0 +1,271 @@
+/*
+    Copyright (C) 2023 Jean Kieffer
+
+    This file is part of Arb.
+
+    Arb is free software: you can redistribute it and/or modify it under
+    the terms of the GNU Lesser General Public License (LGPL) as published
+    by the Free Software Foundation; either version 2.1 of the License, or
+    (at your option) any later version.  See <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef ACB_THETA_H
+#define ACB_THETA_H
+
+#include <stdio.h>
+#include <math.h>
+#include "ulong_extras.h"
+#include "fmpz.h"
+#include "fmpz_mat.h"
+#include "fmpz_lll.h"
+#include "arb.h"
+#include "acb.h"
+#include "acb_poly.h"
+#include "arb_mat.h"
+#include "acb_mat.h"
+#include "acb_modular.h"
```

Comment:
The only needed headers for this header is `fmpz_mat.h` and `acb_types.h`. This needs to be changed. I know that will force someone to go through all sources and tests in order to make it compile, but it is sort of important.